### ISSUE TITLE
[Consistency] User encounters design components that are out of date.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#76954]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Decision Reviews Team
**Product:** Board Appeal (Notice of Disagreement) VA Form 10182
**Feature:** Request an Extension Feature (OMB Approved No. 2900-0674)

## Findings details
**VA.gov Experience Standard - issue:** User encounters design components that are out of date.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the "Need Help" section, the 711 phone link doesn't include "TTY" inside the link text.

## Recommended action
The `va-telephone` component includes a `tty` prop that puts the "TTY" text inside the link text and in the `aria-label` for screen reader users.

## References

- **WCAG Success Criteria:** [WCAG 2.4.9 Link Purpose (Link Only) - Level AAA](https://www.w3.org/TR/WCAG21/#link-purpose-link-only)
- **Modality:** N/A
- **Design System Component:** [Telephone ](https://design.va.gov/components/telephone)
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by rocio-desantiago** [comment]
PR ready for review. 

@rubensun @eileen-coforma 
I'm curious if we want `Need Help` to intentionally look different across our 3 forms:
| HLR | 
| ------- | 
| <img width="600" alt="HLR Need Help footer" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/37054305/2ab48241-e0d4-43ac-8b12-0f64bde21bf3"> |

NOD |
| ------ |
| <img width="600" alt="NOD Need Help footer" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/37054305/61f6da38-bc92-43bc-b2e8-58c654936a6b"> |

| SC |
| ----- |
| <img width="600" alt="SC Need Help footer" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/37054305/607d3fb3-2c42-4699-9ded-fb5cc1abaa62"> |

**Comment by eileen-coforma** [comment]
No. It should look the same. Let me ask about the phone number. One of those numbers lead to a direct line for Benefits, while one of them is the general help line. Not sure why we'd want to direct to the general help if we can go directly to Benefits.

**Comment by HeatherWidmont** [comment]
This is ready to be validated, they will huddle on this today

**Comment by rocio-desantiago** [comment]
Validated with @eileen-coforma today.
